src/ideal/ideal.jl

```diff
@@ -618,7 +618,7 @@ the syzygy module of the previous module, starting with the given
 ideal/module.
 The `max_length` can be set to $0$ if the full free resolution is required.
 """
-function fres(id::Union{sideal{T}, smodule{T}}, max_length::Int, method::String = "complete") where T <: Nemo.RingElem
+function fres(id::Union{sideal{T}, smodule{T}}, max_length::Int, method::String = "complete") where T <: Nemo.FieldElem
```

Comment:
Unfortunately `T` here is the polynomial ring element, which is never going to be a field element.
See https://github.com/oscar-system/Singular.jl/pull/447/checks?check_run_id=2963461194#step:6:224
Something like this is probably the way to go.
```julia
function fres(id::Union{sideal{spoly{T}}, smodule{spoly{T}}}, max_length::Int, method::String = "complete") where T <: Nemo.FieldElem
   ...
   return sresolution{spoly{T}}(R, r, Bool(minimal))
end

```

Reply:
@hannes14 did you see this comment?

Reply:
Yes, I am working on it with this hint.@fingolfin, @tthsqe12